eng/pipelines/runtime-extra-platforms-other.yml

```diff
@@ -103,6 +103,64 @@ jobs:
           eq(dependencies.evaluate_paths.outputs['SetPathVars_coreclr.containsChange'], true),
           eq(variables['isRollingBuild'], true))
 
+#
+# CoreCLR NativeAOT release build (with checked runtime) and additional libraries tests that are known to be passing
+# Only when CoreCLR or library is changed
+#
+- template: /eng/pipelines/common/platform-matrix.yml
+  parameters:
+    jobTemplate: /eng/pipelines/common/global-build-job.yml
+    helixQueuesTemplate: /eng/pipelines/libraries/helix-queues-setup.yml
+    buildConfig: Release
+    platforms:
+    - windows_x64
+    - Linux_arm64
+    jobParameters:
+      testGroup: innerloop
+      isSingleFile: true
+      nameSuffix: NativeAOT_Checked_Libs_Passing
```

Comment:
nit: if there is no `_Failing` counterpart template, should we simplify it and remove `_Passing` from these suffixes?
e.g. coreclr and mono legs also have exclusions but no such suffix was used there.

Reply:
Yes, I haven't exactly been a fan when the suffix was introduced either.